### ISSUE TITLE
Build failure: neovide-0.13.0

### ISSUE THREAD

**Issue by ginkogruen** [NixOS/nixpkgs#314827]
### Steps To Reproduce

1. build `neovide`

### Build log

[Link to build log gist](https://gist.github.com/ginkogruen/4f25e4b73cd4bc5795a120f901b87277)

### Additional context

If I remember correctly the 0.12.2 version was working (but don't rely on my memory).

### Notify maintainers

@ck3d

### Metadata

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
- system: `"aarch64-darwin"`
- host os: `Darwin 23.4.0, macOS 14.4.1`
- multi-user?: `yes`
- sandbox: `no`
- version: `nix-env (Nix) 2.18.2`
- channels(root): `"nixpkgs"`
- nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixpkgs`

```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by MultisampledNight** [comment]
(fwiw I don't maintain neovide anymore and am not able to help with issues on Darwin due to lack of an appropriate machine either, sorry!)

**Comment by ginkogruen** [comment]
Oh sorry my bad I looked at the stable nixpkgs for finding out who is a maintainer.

**Comment by sysedwinistrator** [comment]
This was fixed by https://github.com/neovide/neovide/pull/2587, which is included 0.13.1.
There's already an open PR: https://github.com/NixOS/nixpkgs/pull/315210

The issue is that on macOS, one test is running a login shell via /usr/bin/login, which resets the environment including the PATH. Because of that, the test could no longer find the nvim binary. The fix is to not use a login shell when `TERM` is set (Nix hardcodes `TERM` to `xterm-256color` since 2.3: https://github.com/NixOS/nix/blob/5786e1ae7c300b3c7434e7df99b41f180dc42e37/doc/manual/src/release-notes/rl-2.3.md?plain=1#L28-L31)

**Comment by ginkogruen** [comment]
That's good news :) Will close this as soon as the PR hits `master`.
